PinpointKit/PinpointKit/Sources/Core/Editing/EditImageViewController.swift

```diff
@@ -327,9 +329,14 @@ public final class EditImageViewController: UIViewController, UIGestureRecognize
             let annotationViewIsNotBlurView = !(possibleAnnotationView is BlurAnnotationView)
             
             if let annotationView = possibleAnnotationView {
+                let wasTopmostAnnotationView = (annotationsView.subviews.last == annotationView)
```

Comment:
@mattbischoff No. My justification here is that both operators in the statement use the `=` symbol, and find it more readable than:

```swift
let wasTopmostAnnotationView = annotationsView.subviews.last == annotationView
```

Let me know if you feel strongly about dropping them.